code/game/turfs/simulated/floor/plating.dm

```diff
@@ -269,6 +269,12 @@
 					var/mob/living/live = L.buckled
 					if("lava" in live.weather_immunities)
 						continue
+			if(ishuman(L))
```

Comment:
chasms do the same thing for jaunters, if there's a less snowflake-y method then sure.